scripts/ci/test_pytorch_blade.sh

```diff
@@ -17,11 +17,13 @@ SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 export CXXFLAGS=${CXXFLAGS:-"-Wno-deprecated-declarations"}
 export CFLAGS=${CFLAGS:-"-Wno-deprecated-declarations"}
 export CUDA_HOME=${CUDA_HOME:-/usr/local/cuda}
+export TF_CUDA_HOME=${CUDA_HOME}
```

Comment:
Nit: why is `TF_CUDA_HOME`?

Reply:
Yes, I agree with you. It's `TF_CUDA_HOME` here is because `@cuda_supplement` requires it. `@cuda_suppliment` does so to reuse TF's env var.